### ISSUE TITLE
Feature: World DB snapshot after each squash. (or periodically)

### ISSUE THREAD

**Issue by avarishd** [azerothcore/azerothcore-wotlk#15517]
### Describe your feature request or suggestion in detail

Periodically or after every DB squash, release world DB snapshot.
Similar to Trinity and Cmangos, this would speed work between the open source cores even further. This snapshot will be only used for this, as it will be "outdated" the moment anything core related gets changed.

Export world DB -> Compress (zip) -> Release 

https://github.com/TrinityCore/TrinityCore/releases
https://github.com/cmangos/wotlk-db

**Comment by avarishd** [comment]
I get it, but the idea of this is to have the world DB accessible without having to download or set up anything else, just a straight up .zip containing world db, which can then be manually imported.

**Comment by Kitzunu** [comment]
No. That is just another way for people to screw up installation.
And no way to support it as it will be out dated the day after.
AC will not provide it
